radar/2021-01-01/checkov.md

```diff
@@ -0,0 +1,12 @@
+---
+title:      "Checkov"
+ring:       assess
+quadrant:   platforms-and-aoe-services
+
+---
+
+Checkov is a static code analysis tool for infrastructure-as-code.
+
+It scans cloud infrastructure provisioned using Terraform, Terraform plan, Cloudformation, Kubernetes, detects security and compliance misconfigurations.
```

Comment:
I think a bulletpoint list would be great for the scan features. What do you think?

Reply:
done :)